lib/jnpr/junos/facts/swver.py

```diff
@@ -80,12 +80,15 @@ def __ne__(self, other):
 def _get_swver(dev, facts):
     # See if we're VC Capable
     if facts['vc_capable'] is True:
-        return dev.rpc.cli("show version all-members", format='xml')
-    else:
         try:
-            return dev.rpc.cli("show version invoke-on all-routing-engines", format='xml')
```

Comment:
@shermdog I have not changed that line. https://github.com/Juniper/py-junos-eznc/pull/415/files#diff-d8b197309249b4c226a7a25c6a606de1R88
the way git is showing it with changes makes it feel that way. Please review file in raw format.